### ISSUE TITLE
Accessing to the code source of a homemade dependency via github

### ISSUE THREAD

**Issue by nhenin** [rikvdkleij/intellij-haskell#594]
I'm open sourcing a part of the project of my company now, I have created a public repository on github..
And I use this package that way in `extra-deps` in my `stack.yml` file :
```  
 - github: dolla-consortium/base
    commit: ee600d4b374f5b6aa54d326f4161f84a20acab8e
```
Everything works well, but somehow I can't access to the code source from the project that uses it in Intellij. 

Thanks for the helps in advance :-)

**Comment by nhenin** [comment]
Ok I'm progressing :-) 
![image](https://user-images.githubusercontent.com/471741/88475415-d01a0480-cf2f-11ea-95a0-94a08cfff0b8.png)

**Comment by nhenin** [comment]
Ok I have copied manually the lib and it works, maybe you are not supporting this feature yet ?

**Comment by rikvdkleij** [comment]
> Ok I have copied manually the lib and it works, maybe you are not supporting this feature yet ?

Yes, for libraries only code from hackage is downloaded to the plugin's cache.

Normally when a Stack project has multiple Cabal projects, they are defined in as `packages` in `stack.yaml`. That is supported.

> And by curiosity, how do you use Hoogle with your plugin ?

Hoogle is used to retrieve the Haddock documentation (as in your screenshot) and it can be used for navigation (in menu Navigate/Haskell).

Btw, you didn't mention to location of your repo.

**Comment by nhenin** [comment]
`Normally when a Stack project has multiple Cabal projects, they are defined in as packages in stack.yaml. That is supported.`
I was doing that at the beginning, but when the project get more mature that's a transition you want to do eventually, because now I have 13 sub-project... And some of them are going to be public in github, so I can't do that anymore...
Do you plan to support this ? It's going to become a pb eventually....

`Btw, you didn't mention to location of your repo.`

What do you mean ? this ? 
```
- location:
    git: git@github.com:commercialhaskell/stack.git
    commit: 6a86ee32e5b869a877151f74064572225e1a0398
```
It changed with the new versions of stack (https://stackoverflow.com/questions/38377521/haskell-stack-install-package-dependency-from-github) you don't need it anymore...

**Comment by rikvdkleij** [comment]
> Do you plan to support this ? It's going to become a pb eventually for us....

I will take a look soon. Should be doable.

> What do you mean ? this ?

No, I'm referring to this:
> I'm open sourcing a part of the project of my company now, I have created a public repository on github..

**Comment by nhenin** [comment]
That would be great @rikvdkleij :-) I appreciate the quality of  your support 👍

**Comment by rikvdkleij** [comment]
Yes, issue can stay open.
